### PR TITLE
feat: add redirect link to pre-built providers json file

### PR DESCRIPTION
**Pull request by ansgarm**
In preparation for #2173, blocks #2181

### CODE REVIEW

File: cdk.tf/vercel.json [hunk 1]
```diff
@@ -190,6 +190,10 @@
     {
       "source": "/crash-reporting",
       "destination": "https://www.terraform.io/cdktf/telemetry#crash-reporting"
+    },
+    {
+      "source": "/.well-known/prebuilt-providers.json",
```
Comment (DanielMSchmidt) [inline cdk.tf/vercel.json:195]: any reason for the `.well-known`? I couldn't find the prebuilt-providers.json in the [wiki](https://en.wikipedia.org/wiki/Well-known_URI#:~:text=A%20well%2Dknown%20URI%20is,well%2Dknown%20locations%20across%20servers.)

File: cdk.tf/vercel.json [hunk 2]
```diff
@@ -190,6 +190,10 @@
     {
       "source": "/crash-reporting",
       "destination": "https://www.terraform.io/cdktf/telemetry#crash-reporting"
+    },
+    {
```
Comment (DanielMSchmidt) [inline cdk.tf/vercel.json:194]: Can we add more "this is not for leisure but depended on" context or does that break the build?
```suggestion
    { "//": "This is used by the cdktf provider subcommands"
```

File: cdk.tf/vercel.json (hunk 1 above)
Comment (ansgarm) [reply cdk.tf/vercel.json:195]: I tried to use that to suggest that this is depended on. E.g. the TF Registry uses this as well with something not found in the wiki `https://registry.terraform.io/.well-known/terraform.json` 😁

File: cdk.tf/vercel.json (hunk 2 above)
Comment (ansgarm) [reply cdk.tf/vercel.json:194]: Yeah, such a comment seems reasonable – let's see whether Vercel allows for the `//` attribute.

**Comment by ansgarm** [comment]
^ @DanielMSchmidt adding the comment is not supported by Vercel :/

File: cdk.tf/vercel.json (hunk 1 above)
Comment (DanielMSchmidt) [reply cdk.tf/vercel.json:195]: ok, I see :)

**Pull request merged**